docs/usage.md

```diff
@@ -903,52 +880,106 @@ snpeff_db         = '105'
 snpeff_genome     = 'GRCh38'
 vep_genome        = 'GRCh38'
 vep_species       = 'homo_sapiens'
-vep_cache_version = '106'
+vep_cache_version = '110'
 ```
 
 ### Usage recommendation with AWS iGenomes
 
-Annotation cache is a resource separated from AWS iGenomes, which as its own structure and a frequent update cycle.
-So it is not recommended to put any annotation cache in your local AWS iGenomes folder.
+Annotation cache is a resource separated from AWS iGenomes, which as its own structure and a frequent update cycle, so it is not recommended to put any annotation cache in your local AWS iGenomes folder.
 
-A classical organisation could be:
+A classical organisation on a shared storage area might be:
 
 ```bash
 /data/igenomes/
 /data/cache/ensemblvep
 /data/cache/snpeff
 ```
 
-which can then be used this way in sarek:
+which can then be used this way in Sarek:
 
 ```bash
-nextflow run nf-core/sarek \\
-    --igenomes_base /data/igenomes/ \\
-    --snpeff_cache /data/cache/snpeff/ \\
-    --vep_cache /data/cache/ensemblvep/ \\
+nextflow run nf-core/sarek \
+    --igenomes_base /data/igenomes/ \
+    --snpeff_cache /data/cache/snpeff/ \
+    --vep_cache /data/cache/ensemblvep/ \
     ...
 ```
 
-Or similarly on the cloud:
+Alternatively the data may be stored on AWS S3 storage, therefore the parameters might be:
 
 ```bash
-s3://data/igenomes/
-s3://data/cache/ensemblvep
-s3://data/cache/snpeff
+s3://my-reference-data/igenomes/
+s3://my-reference-data/cache/ensemblvep/
+s3://my-reference-data/cache/snpeff/
 ```
 
 which can then be used this way in sarek:
 
 ```bash
-nextflow run nf-core/sarek \\
-    --igenomes_base s3://data/igenomes/ \\
-    --snpeff_cache s3://data/cache/snpeff/ \\
-    --vep_cache s3://data/cache/ensemblvep/ \\
+nextflow run nf-core/sarek \
+    --igenomes_base s3://my-reference-data/igenomes/ \
+    --snpeff_cache s3://my-reference-data/cache/ensemblvep/ \
+    --vep_cache s3://my-reference-data/cache/snpeff/ \
     ...
 ```
 
 These params can be specified in a config file or in a profile using the params scope, or even in a json or a yaml file using the `-params-file` nextflow option.
 
+### Use pre-populated cache for SNPeff and VEP
+
+Sarek stores a mirror of some cache files on AWS S3 which are set as the default parameters. They contain some genome builds which can be found by checking the contents of the S3 bucket.
+
+```bash
+vep_cache = s3://annotation-cache/vep_cache/
+```
+
+The contents of the VEP cache can be listed with the following command using the S3 CLI:
+
+```bash
+aws s3 --no-sign-request ls s3://annotation-cache/vep_cache/
+```
+
+Simialrly, the SNPeff cache is stored at the following location on S3:
+
+```bash
+snpeff_cache = s3://annotation-cache/snpeff_cache/
+```
+
+The contents can be listed as follows:
+
+```bash
+aws s3 --no-sign-request ls s3://annotation-cache/snpeff_cache
+```
+
+Be sure the specified species, genome and build matches the directory structure within the cache!
```

Comment:
Don't 100% get it here: basically if you download you want to emulate the filder structure you describe above right?